lib/l2meter/emitter.rb

```diff
@@ -197,10 +197,10 @@ def build_token(key, value)
     end
 
     def format_token(key, value)
-      case value
-      when TrueClass
+      case
+      when value == true && !configuration.output_true_false_nil_values?
```

Comment:
Ugh I just realised a side effect of this. When the new behaviour is enabled, this happens:

```ruby
log(:tag1, :tag2) # -> "tag1=true tag2=true"
```

Going to have to think about what to do about it. I guess either we can let it happen (and so add documentation and tests for it) or rewrite the internal logic to prevent it. Or when the option is enabled don't accept bare values at all..?

Reply:
Can we lean on the behavior of other logfmt formatters/loggers to guide us? I'd like to be able to have bare values.

Reply:
I've never seen another logfmt loggers that supports bare values so I don't think there's any prior art.

> I'd like to be able to have bare values.

To clarify: you mean bare values _with_ the new behaviour enabled? That does feel like the best option here, just from the point of view of minimising the behaviour changes. Could make the implementation a lot harder though.

Reply:
> To clarify: you mean bare values with the new behaviour enabled? That does feel like the best option here, just from the point of view of minimising the behaviour changes.

Yes, that. And yeah, implementation could be negatively impacted. Hrmmm...

Reply:
See https://github.com/heroku/l2meter/pull/9/commits/d6d629ad4524e536e259d9d7d678eb44f2c14a72 for a failing test and https://github.com/heroku/l2meter/pull/9/commits/af85c659ff59bb93b59b418cd30b7b5d8a60328f for my proposed fix.